Makefile

```diff
@@ -1,20 +1,24 @@
 TEST_WORKER_COUNT = 8
 
-REPOSITORY = ghcr.io/pinecone-io/canopy
+POETRY_INSTALL_SYSTEM_ARGS = -E cohere -E transformers
+POETRY_INSTALL_EXTRA_ARGS =
```

Comment:
The names here are really confusing, with `SYSTEM_ARGS` being a list of `extra`s, and `EXTRA_ARGS` being used for passing system arguments like `--no-root`...

Let's use something like `DEFAULT_EXTRAS` and `POETRY_INSTALL_ARGS`.